Composer/packages/lib/indexers/src/index.ts

```diff
@@ -69,38 +65,32 @@ class Indexer {
     );
   };
 
-  private getLgImportResolver = (files: FileInfo[], locale: string) => {
-    const lgFiles = files.map(({ name, content }) => {
-      return {
-        id: getBaseName(name, '.lg'),
-        content,
-      };
-    });
-
-    return lgImportResolverGenerator(lgFiles, '.lg', locale);
+  private getResources = (files: FileInfo[], extension: string) => {
+    return files.map(({ name, content }) => ({
+      id: getBaseName(name, extension),
+      content,
+    }));
   };
 
-  public index(files: FileInfo[], botName: string, locale: string, settings: DialogSetting) {
+  public index(files: FileInfo[], botName: string) {
     const result = this.classifyFile(files);
-    const luFeatures = settings.luFeatures;
+    //const luFeatures = settings.luFeatures;
     const { dialogs, recognizers } = this.separateDialogsAndRecognizers(result[FileExtensions.Dialog]);
     const { skillManifestFiles, crossTrainConfigs } = this.separateConfigAndManifests(result[FileExtensions.Manifest]);
     const assets = {
       dialogs: dialogIndexer.index(dialogs, botName),
       dialogSchemas: dialogSchemaIndexer.index(result[FileExtensions.DialogSchema]),
-      lgFiles: lgIndexer.index(result[FileExtensions.lg], this.getLgImportResolver(result[FileExtensions.lg], locale)),
-      luFiles: luIndexer.index(result[FileExtensions.Lu], luFeatures),
-      qnaFiles: qnaIndexer.index(result[FileExtensions.QnA]),
+      lgResources: this.getResources(result[FileExtensions.lg], '.lg'),
```

Comment:
replace hard coded .lg with FileExtensions.lg
(same for the following changes below)